desktop/libs/indexer/src/indexer/templates/importer.mako

```diff
@@ -234,6 +234,17 @@ ${ commonheader(_("Importer"), "indexer", user, request, "60px") | n,unicode }
             <!-- /ko -->
           </div>
 
+          <div data-bind="visible: createWizard.source.inputFormat() == 'localfile'">
+              <form method="post" action="" enctype="multipart/form-data" id="uploadform">
+                <div >
+                    <input type="file" id="inputfile" name="inputfile" accept=".csv">
+                    <input type="button" class="button" value="Upload" id="but_upload">
+                </div>
+                <label for="path" class="control-label"><div>${ _('Path') }</div>
+                  <input type="text" id="file_path" data-bind="value: createWizard.source.path">
```

Comment:
Here I am doing data-binding, (what I understand after reading data-binding is that path value must be updated). But looks like not happening here.
'Formate Guessing ' is depends on [this](https://github.com/cloudera/hue/blob/master/desktop/libs/indexer/src/indexer/templates/importer.mako#L443) and in this createWizard.source.show() depends on the [self.path](https://github.com/cloudera/hue/blob/master/desktop/libs/indexer/src/indexer/templates/importer.mako#L2140)
i.e. how to deal with this.